### PR TITLE
Feature integrate hotjar

### PR DESCRIPTION
**Pull request by adonisjoe**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,10 +6,12 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="shortcut icon" type="image/png" href="img/logo.png" />
 
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
```
Comment (georgebrata) [inline index.html:9]: Why did you add these again?

Comment (adonisjoe) [reply index.html:9]: I had been using a wrong font that is not in the specification. I changed to the specified font sorry.
